src/DssSpell.sol

```diff
@@ -51,121 +59,309 @@ contract DssSpellAction is DssAction {
+                0,                         // eta
+                address(0)                 // mgr
+            )
         );
 
-        // Bump MIP21 Oracle's `val` to 1,250M as WAD (No need to calculate anything since the rate is 0%)
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
-            "RWA007-A",
-             1_250 * MILLION * WAD
+        // VOTE: https://vote.makerdao.com/polling/QmRxNdG7
+        // Phoenix Labs | 2023-04-01 to 2024-04-01 | 120 MKR | 0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA
+        VestLike(MCD_VEST_MKR_TREASURY).restrict(
+            VestLike(MCD_VEST_MKR_TREASURY).create(
+                PHOENIX_LABS_STREAM,       // usr
+                120 * WAD,                 // tot
+                APR_01_2023,               // bgn
+                APR_01_2024 - APR_01_2023, // tau
+                0,                         // eta
+                address(0)                 // mgr
+            )
         );
 
-        // Update the RWA007-A `spot` value in Vat
-        DssExecLib.updateCollateralPrice("RWA007-A");
-
-        // PSM Parameter Normalization
-        // Poll:  https://vote.makerdao.com/polling/QmQ1fYm3#poll-detail
-        // Forum: https://forum.makerdao.com/t/signal-request-set-psm-fees-to-0/10894
-
-        //PSM-USDC-A
-        //Reduce the Fee In (tin) by 1% from 1% to 0%.
-        DssExecLib.setValue(MCD_PSM_USDC_A, "tin", ZERO_BASIS_POINTS);
-        //Increase the Target Available Debt (gap) by 150 million DAI from 250 million DAI to 400 million DAI.
-        DssExecLib.setIlkAutoLineParameters("PSM-USDC-A", 10 * BILLION, 400 * MILLION, 24 hours);
-
-        // PSM-GUSD-A
-        // Reduce the Fee In (tin) by 0.1% from 0.1% to 0%.
-        DssExecLib.setValue(MCD_PSM_GUSD_A, "tin", ZERO_BASIS_POINTS);
-        // Increase the Fee Out (tout) by 0.01% from 0% to 0.01%.
-        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", ONE_BASIS_POINT);
-        // Increase the Target Available Debt (gap) by 40 million DAI from 10 million DAI to 50 million DAI.
-        DssExecLib.setIlkAutoLineParameters("PSM-GUSD-A", 500 * MILLION, 50 * MILLION, 24 hours);
-
-        // PSM-PAX-A
-        // Reduce the Fee Out (tout) by 1% from 1% to 0%.
-        DssExecLib.setValue(MCD_PSM_PAX_A, "tout", ZERO_BASIS_POINTS);
-        // Reduce the Target Available Debt (gap) by 200 million DAI from 250 million DAI to 50 million DAI.
-        // Reduce the Maximum Debt Ceiling (line) by 500 million DAI from 1 billion DAI to 500 million DAI.
-        DssExecLib.setIlkAutoLineParameters("PSM-PAX-A", 500 * MILLION, 50 * MILLION, 24 hours);
+        // ----- Ecosystem Actor Dai Transfers -----
+        // VOTE: https://vote.makerdao.com/polling/Qmbndmkr#vote-breakdown
+
+        // VOTE: https://vote.makerdao.com/polling/QmRxNdG7
+        DssExecLib.sendPaymentFromSurplusBuffer(PHOENIX_LABS_STREAM, 347_100);
+
+        // VOTE: https://vote.makerdao.com/polling/QmV9MR8B
```

Comment:
Need a heading here too

Reply:
these are in the same heading as line 354 (Ecosystem Actor Dai transfers).  For some reason (constitution stuff?), GovAlpha gave me 3 different votes, one for each transfer and one for the heading, so that is what i reflected here.  Let me know if you have a better spacing suggestion